k8s-configuration/post-service/post-db-kubeconfig.yml

```diff
@@ -0,0 +1,77 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: k8s-program
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: posts-db
+  namespace: k8s-program
+spec:
+  selector:
+    app: posts-db
+  ports:
+    - protocol: TCP
+      port: 5432
+  clusterIP: None
+---
+
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: posts-db
+  namespace: k8s-program
+spec:
+  serviceName: posts-db
+  replicas: 1
+  selector:
+    matchLabels:
+      app: posts-db
+  template:
+    metadata:
+      labels:
+        app: posts-db
+    spec:
+      containers:
+        - name: posts-db
+          image: postgres:latest
+          env:
+            - name: POSTGRES_USER
+              value: root
```

Comment:
It is much better and more secure to use the value from k8s Secret.
https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#define-container-environment-variables-using-secret-data